api/admin/rmrf.ts

```diff
@@ -0,0 +1,57 @@
+import { PrismaClient } from '@prisma/client';
+import type { VercelRequest, VercelResponse } from '@vercel/node';
+
+import { IS_LOCAL_ENV } from '../../api-lib/config';
+
+// I probably should not write this
+export default async function handler(req: VercelRequest, res: VercelResponse) {
+  const prisma = new PrismaClient();
+
+  if (!IS_LOCAL_ENV) {
+    res.status(200).json('Coordinape destroyed.'); // JK
+  }
+
+  try {
+    const tables = {
+      feedbacks: prisma.feedbacks,
```

Comment:
no easy way to do a "truncate" with GraphQL?

Reply:
my gut tells me this is going to be simpler moving forward, although prisma is non-prefered.

Reply:
my gut tells me this is going to be simpler moving forward, although prisma is nonpreferred.